utils/measure/runDocker.BAT

```diff
@@ -0,0 +1,13 @@
+@echo off
```

Comment:
As I revisit this becaused you closed this for inactivity, I'll provide a final feedback after having worked a bit more with docker.

There are several ways to use docker.
- Distributed image: yes, you need to copy the files.  Building the image would then be part of the github actions put in place for the project and may use a specific Dockerfile.
  The intent would then be to not fetch the github repository at all.
- Local git clone (for debug/other adjustments),
  - Build the image locally using a Docker file;
  - Run the prebuilt image, but override some contents using a "-volume .:/app" option for instance.
  - These methods could be managed together with a "docker-compose.yaml" file which IMHO avoids OS dependent scripts.
  - Build the image locally - this seemed to be the purpose of the Dockerfile that I was using as it was called "measure_local".

This Dockerfile is already using an image "python:3.8-slim-buster" so the local build is quite limited.  
The intent of the 'runDocker' script is clearly to run it locally.  
In order for the 'runDocker' script to work, the repository has to be locally extracted.

Therefore, the argument that the COPY not needed to build the image as the goal is to run things locally.

Mounting the local directory in the container is proposed in the Docker-compose Getting Started guide: https://docs.docker.com/compose/gettingstarted/#step-5-edit-the-compose-file-to-add-a-bind-mount .

That still allows to use a prebuilt image because even if there is already contents for the '/app' directory,  a mount '.:/app' will override the contents with the local contents.

Reply:
Hi thanks for your feedback! Yes the docker compose approach would be nice fit to make it easy to get the container build and running locally for development/tweaking purposes. Also this would eliminate the need for OS dependent scripts and use the tooling of docker ecosystem.
When you have some time on hand, you could create another PR if you want with this approach and I will happily merge it.

Reply:
Well, I don't have a lot of time, this was mainly intended as feedback.